src/a11y/a11y.test.js

```diff
@@ -48,15 +48,15 @@ test('regular tree has no a11y exceptions', async t => {
   const { tree } = t.context
   const component = mountToDoc(
     <div>
-      <DropdownTreeSelect data={tree} showDropDown texts={{ label: 'test' }} />
```

Comment:
@ellinge Regarding the failing tests: `showDropDown` (note the casing) was never a valid prop. So these tests were passing earlier (I think that's the reason why). Including the correct `showDropdown` prop throws a weird a11y error which I've no idea why.

Reply:
Guess I figured it out. https://www.w3.org/TR/wai-aria-1.1/#list & https://www.w3.org/TR/wai-aria-1.1/#tree do not support `aria-readonly`. 

Also, in case of `simpleSelect`, the `role=list` seems more fitting as [`listbox`](https://www.w3.org/TR/wai-aria-1.1/#listbox) implies multiple items can be selected:

> A widget that allows the user to select one or more items from a list of choices.

Reply:
Scratch that. According to https://www.w3.org/TR/wai-aria-1.1/#select, listbox is the correct attribute. Damn, this Aria stuff is confusing.

The only failing test is for radio select now. Gonna look into that next.

Reply:
Added a role="radiogroup" on the dropdown content for radioselect now

Reply:
Based on https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html

Reply:
> What a miss on my part with the casing. Wouldn't have happened with typescript 😁

I see what you did there! :)

How would TS know it's not a valid prop?

Reply:
It would not compile and the IDE would tell me straight away.
Well at least as long as the props interface wasn’t simply `any` or allowing custom props through a any indexer.

Probably can install something in VS Code to check against propTypes as well.
https://marketplace.visualstudio.com/items?itemName=OfHumanBondage.react-proptypes-intellisense

Reply:
> Well at least as long as the props interface wasn’t simply any or allowing custom props through a any indexer.

I think you can enforce something similar via prop types too but not sure. Way back, we built https://github.com/dowjones/react-json-schema-proptypes which would enforce that too (it used a JSON schema and provided prop types in return). 

React components and JS by nature, do not explicitly disallow any custom props. In fact, disallowing them is more work and allowing them sometimes has unintended benefits to the user. In cases like this though, it bites us back. 

But to be fair, it escaped my eyes too. I did see those errors before but I simply thought it was due to bad merge resolutions. So doubly bad for me ignoring things twice. :)

That extension seems like a good one. Gonna try it.